pkg/cmd/taskrun/logs.go

```diff
@@ -39,6 +41,45 @@ type LogOptions struct {
 	AllSteps bool
 }
 
+func logCommand(p cli.Params) *cobra.Command {
+	opts := LogOptions{}
+	eg := `
+	# show the logs of TaskRun named "foo" from the namesspace "bar"
+	  tkn taskrun logs foo -n bar
+	 `
+	c := &cobra.Command{
+		Use:          "logs",
+		Short:        "Show taskruns logs",
+		Example:      eg,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
```

Comment:
Perhaps we should make a `cli.RequiredArg("taskrun", "name of the taskrun")` so that when the validation fails it produces 
```
ERROR: Missing a required argument taskrun: name of the taskrun
```

Reply:
If we want it specific to taskrun logs then we can write it in same taskrun pkg.
Or
May be we can make it more generic writing it in cli pkg which could be used by other subcommands as well.

Reply:
yeah, I think the arg validation can be generic hence `cli.RequiredArg(...)`

Reply:
We can take this in the next iteration, as this could be used by others as well.